skins/oasis/modules/templates/UserPagesHeader_BlogListing.php

```diff
@@ -1,16 +1,12 @@
 <div id="WikiaUserPagesHeader" class="WikiaUserPagesHeader WikiaBlogListingHeader">
-<?php
-	// render edit button / dropdown menu
-	if (!empty($actionButton)) {
-	echo F::app()->renderView('MenuButton', 'Index', array(
-				'action' => $actionButton,
-				'image' => MenuButtonController::BLOG_ICON,
-				'name' => 'createblogpost',
-				));
-	}
-?>
-
-	<h1><?= htmlspecialchars($title) ?></h1>
-	<h2><?= htmlspecialchars($subtitle) ?></h2>
-
+<?php if ( !empty( $actionButton ) ): // render edit button / dropdown menu ?>
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)